CMSgov/mdct-mcr
# fix rule of hooks issue (+0.1%)

## Description

There was a bug introduced in `eb055df` where we were violating the rule of hooks by conditionally rendering the AdminBanner provider. This PR fixes that by moving the provider post-auth.

![Screen Shot 2022-05-26 at 18 46 18](https://user-images.githubusercontent.com/27705928/170601486-c791ae99-f49c-4f49-84a2-763b4fb88058.png)

### How to test

1. Pull and run. The error should not appear in the console. Everything should still work.

### Changed Dependencies

No changed deps.

## Code author checklist

## Reviewer checklist (two different people)
